style.css

```diff
@@ -25,17 +26,93 @@ hr {
   border-top: 3px solid #333;
 }
 
+/* Header Styling */
+
+header {
+  width: 100%;
+}
+
+.main-nav {
+  background-color: #fff;
+  box-shadow: 0 1px 15px rgb(97, 97, 97);
+}
+
+#books,
+#contact,
+#current-time,
+footer p,
+#page-nav {
+  width: 90%;
+  margin: 0 auto;
+}
+
+#books,
+#contact,
+#form-section {
+  transition: 400ms ease-in-out;
+}
+
+#page-nav {
+  height: 40px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+
+#contact a,
+#page-nav span a {
+  text-decoration: none;
+  color: #333;
+  transition: 250ms ease-out;
+}
+
+#page-nav span a {
+  font-weight: bold;
+  font-size: 15px;
+}
+
+#page-nav ul li a {
+  font-size: 14px;
+  text-decoration: none;
+  transition: 300ms ease-in-out;
+  color: #333;
+}
+
+#contact a:hover,
+#page-nav li a:hover,
+#page-nav span a:hover {
+  color: blue;
```

Comment:
- [ ] OPTIONAL: In addition to hover styling you can also make the focused/clicked `navLink` have a different color/styling for a better user experience. You did great overall! :clap: 
![image](https://user-images.githubusercontent.com/64894927/129345037-240e942b-1441-4e57-918b-75f9869a4354.png)
![image](https://user-images.githubusercontent.com/64894927/129345104-7ba23921-9b23-4afb-86f8-0c77d4006037.png)

Reply:
Hi @henatan99
Thank you very much for this.
I'll fix it